ggml/src/ggml-vulkan/vulkan-shaders/im2col.comp

```diff
@@ -1,6 +1,11 @@
 #version 450
 
 #extension GL_EXT_shader_16bit_storage : require
+#extension GL_EXT_spirv_intrinsics: enable
+
+#if RTE16
+spirv_execution_mode(capabilities = [4467], 4462, 16); // RoundingModeRTE, 16 bits
```

Comment:
Is this the only way to do this? No support in GLSL?

I've planned looking into intrinsics as well since there's also an extension for DP4A (`VK_KHR_shader_integer_dot_product`) that never got support in GLSL as far as I know.

Reply:
Yeah, I didn't see any real support for this in glslang.

There was a spec proposal for integer dot product (https://github.com/KhronosGroup/GLSL/pull/195) but the glslang implementations were never finished. It was partially implemented, though I don't remember how far away from being complete it is.